### PR TITLE
Akamas doc v1.1.0

### PR DESCRIPTION
**Pull request by strmatteo**

### CODE REVIEW

File: akamas/1.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+1. Create a file called `triggered-event.json` with the following contents:
+    ```json
+    {
+    "type": "sh.keptn.event.mystage.mysequence.triggered",
+    "specversion":"1.0",
+    "source":"manual-trigger",
+    "data": {
+      "project":"myproject",
+      "stage":"mystage",
+      "service":"myservice",
```
Comment (christian-kreuzberger-dtx) [inline akamas/1.1.0/README.md:88]: remove the trailing comma

Comment (strmatteo) [reply akamas/1.1.0/README.md]: done, thanks

File: akamas/1.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    <details>
+    <summary>How to install the Akamas container</summary>
+
+    - Make an application directory such as /usr/local/keptn
+    - In the application directory, download the files below.
+      ```bash
+      curl https://akamas.s3.us-east-2.amazonaws.com/integrations/keptn/docker-compose.yml -o docker-compose.yml
+      curl https://akamas.s3.us-east-2.amazonaws.com/integrations/keptn/env.templ -o env.templ
+      ```
+    - Edit the `docker-compose.yml` file by specifying the certificate folder and corresponding files in order to expose the integration API called by Keptn webhook via HTTPS. This is the same folder specified when enabling HTTPS on Akamas.
```
Comment (TannerGabriel) [inline akamas/1.1.0/README.md:20]: Could you also mention that the Dockerfile expects the certificates to have a specific name (e.g. `akamas.pem` and `akamas.key` - see `docker-compose.yaml`)

File: akamas/1.1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+2. Port 6000 must be open on the Akamas server. 
+3. Install the Akamas pre-built container. This container will act as a bi-directional bridge between Akamas Studies and Keptn webhooks. 
+
+    <details>
+    <summary>How to install the Akamas container</summary>
+
+    - Make an application directory such as /usr/local/keptn
+    - In the application directory, download the files below.
+      ```bash
+      curl https://akamas.s3.us-east-2.amazonaws.com/integrations/keptn/docker-compose.yml -o docker-compose.yml
```
Comment (TannerGabriel) [inline akamas/1.1.0/README.md:17]: The container name `keptn-webhook` is not very specific. Maybe change it to something like `keptn-akamas-webhook-service`.

File: akamas/1.1.0/README.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+      curl https://akamas.s3.us-east-2.amazonaws.com/integrations/keptn/env.templ -o env.templ
+      ```
+    - Edit the `docker-compose.yml` file by specifying the certificate folder and corresponding files in order to expose the integration API called by Keptn webhook via HTTPS. This is the same folder specified when enabling HTTPS on Akamas.
+    - Rename the `env.templ` file to `.env`, then edit the file by replacing the values of the `KEPTN_URL` and `KEPTN_TOKEN` variables with your own values.
```
Comment (TannerGabriel) [inline akamas/1.1.0/README.md:21]: Why not just save it as `.env` instead in the `curl` command above?

**Review by TannerGabriel — CHANGES_REQUESTED** [review]
Nice integration. There are some minor things that should be fixed before merging.

File: akamas/1.1.0/README.md (hunk 2 above)
Comment (strmatteo) [reply akamas/1.1.0/README.md:20]: done, thanks

File: akamas/1.1.0/README.md (hunk 3 above)
Comment (strmatteo) [reply akamas/1.1.0/README.md:17]: done

File: akamas/1.1.0/README.md (hunk 4 above)
Comment (strmatteo) [reply akamas/1.1.0/README.md:21]: done, thanks

**Review by strmatteo — COMMENTED** [review]
changes applied

**Review by TannerGabriel — APPROVED** [review]
LGTM

**Pull request merged**